setup.py

```diff
@@ -50,5 +49,7 @@
         "requests",
         "ddtrace",
         "importlib-metadata; python_version < '3.8'",
```

Comment:
```suggestion
        "importlib-metadata>=1.4; python_version < '3.8'",
```
1.4+ and above is for python3.8+. Also this depends on what indexd is using it for. Can you do a scan to see if indexd uses it for, else I think we should remove and let pip-compile resolve it, same for typing extension

Reply:
used here: https://github.com/NCI-GDC/indexd/blob/develop/indexd/index/blueprint.py#L520

do you think we should remove the endpoint?